action.yml

```diff
@@ -65,6 +65,10 @@ inputs:
   headed:
     description: 'Whether or not to use headed mode'
     required: false
+  publish-summary:
+    description: 'Whether or not to publish job summary'
+    required: false
+    default: true
```

Comment:
```suggestion
    default: false
```

Reply:
This change would make the PR into a BREAKING CHANGE which would require a new major version to be released. I'm not sure whether this is necessary or justifiable, unless there are other related changes taking place.